File: cpp/day23.cpp
#include<bits/stdc++.h>

using namespace std;
string toBinary(int n){
    string bin="";
    while(n/2!=0){
        if(n%2==0)
            bin+='0';
        else
            bin+='1';
        n/=2;
    }
    if(n%2==1)
        bin+='1';
    return bin;

}
int main(){
    int n;
    cin>>n;
    string bin = toBinary(n);
    reverse(bin.begin(),bin.end());
    cout<<bin;
}

File: python/day23.py
def tobin(n):
    bin = "";
    while(n/2!=0):
        if n%2==0:
            bin = bin + "0"
        else:
            bin = bin + "1"
    if n%2==1:
        bin = bin + "1"
    return bin

n = int(input())
bin = tobin(5)
print(bin)
